<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>751</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Page_0">
     <widget class="QPushButton" name="prim">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Gestion des matieres primaires</string>
      </property>
     </widget>
     <widget class="QPushButton" name="prod">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>160</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Gestion des produits</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Page_1">
     <widget class="QPushButton" name="Aff">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher les matires primaires</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Ajout">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>190</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter les matieres primaires</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Page_2">
     <widget class="QPushButton" name="Ajout1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter un produit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Aff1">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>220</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher un produit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Page_3">
     <widget class="QLabel" name="Id">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Id</string>
      </property>
     </widget>
     <widget class="QLabel" name="Nom">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="Prix">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Page_4">
     <widget class="QLabel" name="Id1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Id</string>
      </property>
     </widget>
     <widget class="QLabel" name="Nom1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="Prix1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prix">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>270</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Page_5">
     <widget class="QPushButton" name="SupProd">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>170</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer un produit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Modprod">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>170</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier un produit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Page_6">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier matiere premiere</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>200</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer matiere premiere</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
